packages/addons/service/nextpvr/package.mk

```
@@ -25,4 +25,9 @@ addon() {
 
 post_install_addon() {
   sed -e "s/@NEXTPVR_VERSION@/${PKG_ADDON_VERSION}/g" -i "${INSTALL}/bin/nextpvr-downloader"
+
+  mkdir -p ${INSTALL}/bin
+  cp $(get_install_dir libmediainfo)/shared-lib/libmediainfo.so ${INSTALL}/bin
+  cp $(get_build_dir libhdhomerun)/hdhomerun_config ${INSTALL}/bin
+
```

.so file should not be in LE PATH. I also don't think hdhomerun_config should be added to the LE PATH, as this addon is nextpvr. Place it in the nextpvr-bin directory or in an lbin directory?
```suggestion

  mkdir -p ${INSTALL}/lib.private
  cp $(get_build_dir libmediainfo)/.libs/libmediainfo.so ${INSTALL}/lib.private
  cp $(get_build_dir libhdhomerun)/hdhomerun_config ${INSTALL}/bin

```

---

I actually thought nextpvr-bin was in the $PATH but I see I misread that /and I will be able to move the library from bin.

---

OK I had to make a slight change since .libs is not directly under the build directory.